Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3-slim
+
+ENV PROJECT_ROOT /app
+WORKDIR $PROJECT_ROOT
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+COPY . .
+CMD python3 django-reference-app/manage.py runserver 0.0.0.0:8000
```

Comment:
might want to add ```EXPOSE``` and/or ENV var to specify Django port

Reply:
We're going to break this out into a separate, (much) smaller task so we can get this PR merged with as little additional work as possible.